test/users.js

```diff
@@ -0,0 +1,143 @@
+const chai = require('chai');
+const { expect } = chai;
+const sinon = require('sinon');
+const request = require('supertest');
+const { User } = require('../models');
+const sandbox = sinon.createSandbox();
+const rewire = require('rewire');
+
+const RoleMiddleware = require('../middlewares/verify-role');
+
+describe('User', () => {
+
+  beforeEach(() => {
+    fakeAuth = (req, res, next) => next()
+    authStub = sandbox.stub(RoleMiddleware, 'isAdminRole').callsFake(fakeAuth);
+    authOwnerStub = sandbox.stub(RoleMiddleware, 'isOwner').callsFake(fakeAuth);
+
+    app = rewire('../app');
```

Comment:
nit: No es necesario ese rewire en cada it ni al terminar.

Reply:
@soyarielruiz lo probé sin el rewire y me falla uno de los tests, por qué será?